eva_cttv_pipeline/evidence_string_generation/evidence_strings.py

```diff
@@ -28,50 +32,37 @@ def get_cttv_variant_type(clinvar_record_measure):
     return cttv_variant_type
 
 
-def process_clinical_significance(clin_sig):
-    """
-    Processes ClinVar clinical significance string into a format suitable for OT JSON schema.
-
-    Namely, splits multiple clinical significance levels into an array and normalises names (to lowercase, using only
-    spaces for delimiters). Multiple levels of clinical significance are separated using two delimieters: ('/', ', ').
-    See /clinvar-variant-types/README.md for further explanation. The output array is sorted alphabetically.
-
-    Example: 'Benign/Likely benign, risk_factor' → ['benign', 'likely benign', 'risk factor'].
-    """
-    return sorted(re.split('/|, ', clin_sig.lower().replace('_', ' ')))
-
-
 class CTTVEvidenceString(dict):
-    """
-    Base evidence string class. Holds variables and methods common between somatic and genetic
-    evidence strings.
-    Subclass of dict to use indexing.
-    """
-
-    def __init__(self, a_dictionary, clinvar_record=None, ref_list=None,
-                 ensembl_gene_id=None, report=None, trait=None):
-        super().__init__(a_dictionary)
+    """Base evidence string class. Holds variables and methods common between somatic and genetic evidence strings.
+    Subclass of dict to use indexing."""
 
-        if ensembl_gene_id:
-            self.add_unique_association_field('gene', ensembl_gene_id)
-        if clinvar_record:
-            self.add_unique_association_field('clinvarAccession', clinvar_record.accession)
+    def __init__(self, a_dictionary, clinvar_record, clinvar_trait, ontology_id, ontology_label, ensembl_gene_id):
```

Comment:
Can "a_dictionary" be renamed to something like "base_json_key_value_pairs"?

Reply:
Yes, I agree, `a_dictionary` is another one of those super ancient names which survived to our days. I chose the new name as `base_json_dict`. Fixed: https://github.com/EBIvariation/eva-opentargets/pull/167/commits/7b455c2a2b53219920b7c691eabe2c3d3971ff2c